websites/docs-smoke-test/src/releases/excerpt-rendering-test.mdx

```diff
@@ -0,0 +1,82 @@
+---
+date: 2000-02-02
+title: Test Excerpt Rendering
+description: |
+  This release exposes the most common Markdown syntax before and after the read more separator to test rendering
+type: fix
+topics:
+  - Testing
+  - Rendering
+---
+
+<Subtitle>Subtitle Text</Subtitle>
+
+A plain paragraph `with inline code` in it. And **bold** and *italic*.
+
+# Heading 1
+
+ * Bullet List
+ * Bullet List 2
+
+ 1. Ordered List 1
+ 2. Ordered List 2
+
+## Heading 2
+
+> A blockquote piece
+
+```sh
+Code in a fenced code block
+```
+### Heading 3
+
+| table head 1 | table head 2 |
+|---|---|
+| table call 1A | table cell 2A |
+| table cell 2A | table cell 2B |
+
+<dl>
+  <dt>Term</dt>
+  <dd>Definition of the Term</dd>
+</dl>
+
+separator line below this
+
+----
+
+<!--more-->
+
+----
```

Comment:
So is this format how you want people to use the "write more"?

Feels a bit fragile, maybe it's just my impression.

Reply:
We went through that decisionmaking in a PR that was explicitly about agreeing on the format specification. 

That format is the current format (but that could be migrated).   One requirement is that it must be handled opaque by Markdown preview implementations, so it must be either a valid but invisible markdown element or an HTML comment.  

I proposed a different format, too (e.g. just a horizontal line) but the tech writers preferred this format becuause it's verbally explicit.

Reply:
Thanks for the heads up.